.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: rust
 
 rust:
-  - nightly-2018-07-24  # pinned toolchain for clippy
-  - 1.26.0              # minimum supported toolchain
```

Comment:
Instead of dropping this, it would be better to update to a recent stable. If we can still support `1.31.0` I'd be happier, but otherwise `1.37.0` would be fine.